plugins/woocommerce/includes/admin/settings/class-wc-settings-payment-gateways-react.php

```diff
@@ -71,6 +72,12 @@ public function preload_settings( $settings ) {
 			return $settings;
 		}
 
+		$user_nox_meta = get_user_meta( get_current_user_id(), Payments::USER_PAYMENTS_NOX_PROFILE_KEY, true );
```

Comment:
Let's not do this here now that we have a better place [here](https://github.com/woocommerce/woocommerce/blob/a0fe982a5eed5e8cfe3b173dcfdf39ac97adc2f0/plugins/woocommerce/src/Internal/Admin/Settings/PaymentsController.php#L113-L113). There we have access to the Payments service that we can use to call to get the location country rather than reading the user meta directly.

Reply:
Fixed in 6913909d869169d541f7ad4a7abb3739cf375b84 🙇